modoboa/core/commands/templates/settings.py.tpl

```diff
@@ -295,6 +295,10 @@ LOGGING = {
     }
 }
 
+SILENCED_SYSTEM_CHECKS = [
+    "security.W019",  # modoboa uses iframes to display e-mails
+]
+
```

Comment:
This setting is already defined in amavis defaut settings so, when this plugin is enabled, it won't work. (see https://github.com/modoboa/modoboa-amavis/blob/master/modoboa_amavis/settings.py#L7)
I think we can use something like this: https://github.com/modoboa/modoboa-contacts/blob/master/modoboa_contacts/settings.py#L15

Reply:
Yeah I noticed that before, I'll submit a PR for modoboa-amavis

Reply:
Great!